src/apiCalls.js

```diff
@@ -1,62 +1,35 @@
 let fetchCustomers = () => {
   return fetch("http://localhost:3001/api/v1/customers")
-  .then(response => response.json())
+    .then(response => response.json())
```

Comment:
removed the .then(data => etc over into the scripts file so that way I could DO something with it IN THAT MOMENT. ***Lesson of the day. Awesome.